connect/cli/plugins/project/helpers.py

```diff
@@ -0,0 +1,56 @@
+#  Copyright © 2021 CloudBlue. All rights reserved.
+
+from cookiecutter.main import cookiecutter
+from cookiecutter.config import DEFAULT_CONFIG
+from cookiecutter.utils import rmtree
+from cookiecutter.exceptions import OutputDirExistsException
+
+from connect.cli.plugins.project.constants import (
+    AVAILABLE_PROJECTS,
+    BOILERPLATE_URL,
+)
+
+import glob
+import json
+
+from cmr import render
+
+import click
+from click import ClickException
+
+from connect.reports.datamodels import RepositoryDefinition
+
+
+def bootstrap_project(data_dir: str):
+    click.secho('Bootstraping report project...\n', fg='blue')
+
+    # Avoid asking rewrite clone boilerplate project
+    cookie_dir = DEFAULT_CONFIG['cookiecutters_dir']
+    rmtree(cookie_dir)
+    try:
+        project_dir = cookiecutter(BOILERPLATE_URL, output_dir=data_dir)
+        click.secho(f'\nReport Project location: {project_dir}', fg='blue')
+    except OutputDirExistsException as error:
+        raise ClickException(str(f'\n{error}'))
```

Comment:
Maybe lets add a more human readable error, "already exists a project directory called XXX from one of your existing projects, if you would like to use same name delete the directory first."

Reply:
"The directory XXXX is already created, if you would like to use that name, please delete the directory or use another location."
What do you think @marcserrat ?